src/disable_auto_logout.js

```diff
@@ -1,23 +1,32 @@
-(function () {
+import { isScript } from './startup/startup.js'
+import { sendBackgroundScriptAMessage, MESSAGE_TYPE } from './coms/communication.js';
+
+const init = () => {
   var isEnabled = false;
 
-  const intervalID = setInterval(() => {
+  setInterval(() => {
     if (!isEnabled) {
       return;
     }
 
+    // There is probably a better way to do this, utilizing something like
+    // angular.element($('[ng-controller]')).scope().<something>.clearIdle()
+    // But I couldn't figure it out. And hey, this works!
     window.dispatchEvent(new KeyboardEvent('keyup', null));
```

Comment:
Maybe worth adding a comment on what this is specifically attempting to accomplish? When we're on the border of reverse engineering like this I find it useful to figure out what something was attempting to do when it inevitably breaks because of changes to the thing being reverse engineered.

Reply:
Do you mean what the `angular.element` is trying to accomplish? Or what I'm trying to accomplish with the `KeyboardEvent`?

Reply:
The latter. While I generally understand that this is intended to prevent logouts and make the page happy, I'm uncertain why `keyup` is the mechanism to do so. Where did that information come from? You've documented a possible better way which is also helpful.